### DISCUSSION TITLE
Control of row selection by several checkboxes

### DISCUSSION THREAD

**Discussion by AnnaKuhir** [TanStack/table#3549] [General]
Good evening! Thank you for your work, I am actively using this library and have a very interesting question.

Do we have the ability to control the selection of a row not with one checkbox, but with several (I attach a sample picture of how it might look)
And so that at the same time we return an array with approximately the following data

[
    {
        id: checkboxId1
        selectedElement: {selectedElementObj}
    }
    {
        id: checkboxId2
        selectedElement: {selectedElementObj}
    }
]

![image](https://user-images.githubusercontent.com/58226384/142234232-a0525e33-6cd9-490d-afbd-22c868a515fe.png)

**Comment by SimaAmini** [comment]
@AnnaKuhir Did you find a solution?
